easybuild/easyconfigs/p/Python/Python-2.7.6-goolf-1.5.14.eb

```diff
@@ -0,0 +1,115 @@
+name = 'Python'
+version = '2.7.6'
+
+homepage = 'http://python.org/'
+description = """Python is a programming language that lets 
+you work more quickly and integrate your systems more effectively."""
+
+toolchain = {'name': 'goolf', 'version': '1.5.14'} 
+toolchainopts = {'pic': True, 'opt': True, 'optarch': True}
+
+numpyversion = '1.8.1'
+scipyversion = '0.13.3'
+
+source_urls = ['http://www.python.org/ftp/%(namelower)s/%(version)s/']
+sources = [SOURCE_TGZ]
+
+dependencies = [
+    ('bzip2', '1.0.6'),        # for bz2 package
+    ('zlib', '1.2.8'),
+    ('libreadline', '6.2'),
+    ('ncurses', '5.9'),
+    ('LLVM', '3.2'),           # for Numba
+#   ('OpenSSL', '1.0.1f'),     # OS dependency should be preferred for security reasons
+    ('libjpeg-turbo','1.3.0'), # for PIL/Pillow
+    ('libpng','1.6.10'),       # for PIL/Pillow
+    ('freetype', '2.5.3'),     # for PIL/Pillow
```

Comment:
we try to stay away from adding dependencies for specific Python packages; ideally, these should be installed via a separate easyconfig